tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist =
+    py{27,33,34,35,36,37}
```

Comment:
yeah maybe I should look at testing at least 34 and 35 on my projects, though I try not to use the cutting edge stuff